docs/src/tutorial-real-code.md

```diff
@@ -73,12 +73,21 @@ A first proof will likely start in the following form:
 
 Running Kani on this simple starting point will help figure out:
 
-1. What unexpected constraints might be needed on your inputs to avoid "expected" failures.
-2. Whether you're over-constrained. Check the coverage report using `--visualize`. Ideally you'd see 100% coverage, and if not, it's usually because now you've over-constrained the inputs.
+1. What unexpected constraints might be needed on your inputs (using `kani::assume`) to avoid "expected" failures.
+2. Whether you're over-constrained. Check the coverage report using `--visualize`. Ideally you'd see 100% coverage, and if not, it's usually because you've assumed too much.
```

Comment:
I like the previous wording that refers to "over-constraining". So perhaps add that at the end?
```suggestion
2. Whether you're over-constrained. Check the coverage report using `--visualize`. Ideally you'd see 100% coverage, and if not, it's usually because you've assumed too much (thus over-constraining the inputs).
```